workflow/main.py

```diff
@@ -0,0 +1,213 @@
+'''
+Main driver code for end to end TPS workflow
+
+@Author Adam Shedivy 
+
+'''
+
+from visualization_utilities import vis, find_path, process_exists
+from table_generation import PrepTemporalCytoscapeTPS
+from table_refactor import refactor_col_delim_heat, refactor_col_delim
+from requests.exceptions import ConnectionError as CE
+from json.decoder import JSONDecodeError
+import os
+import pandas as pd
+import numpy as np
+from datetime import date
+import subprocess
+import sys
+import time
+import csv
+
+
+def main():
+    '''
+    Outine:
+        - run TPS
+        - gather input data
+        - prepare annotations file
+        - run visualization
+
+    '''
+
+    today = date.today()
+
+    # * Run TPS!
+
+    # * create Base directory
+    baseDir = os.path.abspath(os.path.join('..'))
+    tps_in_dir = os.path.join(baseDir, 'data', 'timeseries')
+    tps_out_dir = os.path.join(
+        baseDir, 'results', 'results_reproduce_egfr_tps_' + str(today))
+    if not os.path.exists(tps_out_dir):
+        os.makedirs(tps_out_dir)
+        print('Created {}'.format(tps_out_dir))
+    print(baseDir)
+    print(tps_in_dir)
+    print(tps_out_dir)
+
+    # * Move files into directory
+    output_sif_orig = os.path.join(baseDir, 'output.sif')
+    activity_tsv_orig = os.path.join(baseDir, 'activity-windows.tsv')
+    output_dest = os.path.join(baseDir, 'workflow', 'output.sif')
+    activity_dest = os.path.join(baseDir, 'workflow', 'activity-windows.tsv')
+
+    os.replace(output_sif_orig, output_dest)
```

Comment:
Why do we need to move files?

Reply:
this was originally to have the output files in the workflow directory, but I changed in so all files produced from TPS to visualization output files are moved to a results folder

Reply:
We discussed how TPS outputs files in the current directory.  We want those files to be located in the same directory as the rest of the output files from this workflow.